src/lib/docs/AccountKit/LinkAuthMethod.md

```diff
@@ -0,0 +1,30 @@
+---
+title: LinkAuth (Method)
+description: Used to generate an action that creates a dependency between a smart contract action and a permission level.
+category: AccountKit
+published: true
+order: 9
+---
+
+# Linkauth
+
+The `linkAuth` method provides a way for an Antelope account owner to require a specific account permission level when executing a specific action on a smart contract.
+## Usage
+
+The `linkAuth` method is available on any [Account](/docs/account-kit/account) instance. Here is an example of how to use it:
+
+```typescript
+const action = testAccount.linkAuth('eosio.token', 'transfer', 'active');
```

Comment:
Using `active` here doesn't make much sense, since you would never use linkauth on an active permission

Reply:
True! I'll use `claim` instead 👍